docs/data-sources/ec_deployment.md

```diff
@@ -32,6 +32,7 @@ These will not be available for interpolation.
 * `deployment_template_id` - ID of the deployment template used to create the deployment.
 * `traffic_filter` - Traffic filter block, which contains a list of traffic filter rule identifiers.
 * `observability` Observability settings. Information about logs and metrics shipped to a dedicated deployment.
+* `tags` Key value map of arbitrary string tags..
```

Comment:
I think you forgot to move this line above the observability one 😄

Reply:
Indeed. I just mashed the "commit suggestion" button.
Fixed in https://github.com/elastic/terraform-provider-ec/pull/244/commits/ea5f851dc733a60302b5beae986c59f97b724e6a